docs/source/getting-started/command-line-interface/run.rst

```diff
@@ -63,3 +63,21 @@ To start the backend server and skip the process cleanup::
 
```

Comment:
This file has been renamed to `backend.rst` to update the new command name; just c/p'ing the relevant docs over to the new file should be fine.